src/core/CONVENTIONS.md

```diff
@@ -332,7 +332,7 @@ import { SavedObjectsType } from 'src/core/server';
 export const myType: SavedObjectsType = {
   name: 'my-type',
   hidden: false,
-  namespaceAgnostic: true,
+  namespaceType: 'agnostic',
```

Comment:
```suggestion
  namespaceType: 'single',
```

I'm not sure why the previous example was setting `namespaceAgnostic` to true, as this really should be the exception compared to the norm. I'd prefer that we change this to `namespaceType: 'single'` to make this more clear...

Reply:
That's super good to know actually! And I was definitely mindlessly copying/pasting this for my plugin so I'll go ahead and change that there as well haha.